openedx/core/openapi.py

```diff
@@ -0,0 +1,20 @@
+"""
+Open API support.
+"""
+
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Open edX API",
+        default_version="v1",
+        description="APIs for access to Open edX information",
+        #terms_of_service="https://www.google.com/policies/terms/",         # TODO: Do we have these?
+        contact=openapi.Contact(email="oscm@edx.org"),
```

Comment:
Can we change this email address to `bom-squad@edx.org`?  We should update openedx.yaml in edx-platform to have the same value.